charts/topolvm/Chart.yaml

```diff
@@ -6,6 +6,7 @@ name: topolvm
 description: Topolvm
 version: 13.0.1
 appVersion: 0.25.0
+kubeVersion: "1.25-1.27"
```

Comment:
We don't use kubeVersion to allow users to use topolvm with unsupported k8s versions at their own risk. It's intentional.